### DISCUSSION TITLE
DisplayLink not working :(

### DISCUSSION THREAD

**Discussion by gshearer** [hyprwm/Hyprland#5588] [Q&A] [answered]
System logs look ok when I plug the panel, but hyprctl monitors only shows my laptop's built-in screen.
Any ideas? BTW this works with Xorg and AwesomeWM.

[  334.728678] usb 7-1: new high-speed USB device number 5 using xhci_hcd
[  334.889179] usb 7-1: New USB device found, idVendor=0bda, idProduct=5412, bcdDevice= 1.25
[  334.889186] usb 7-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[  334.889188] usb 7-1: Product: 4-Port USB 2.0 Hub
[  334.889189] usb 7-1: Manufacturer: Generic
[  334.937528] hub 7-1:1.0: USB hub found
[  334.939385] hub 7-1:1.0: 2 ports detected
[  335.298397] usb 7-1.1: new full-speed USB device number 6 using xhci_hcd
[  335.409242] usb 7-1.1: not running at top speed; connect to a high speed hub
[  335.433134] usb 7-1.1: New USB device found, idVendor=0bda, idProduct=5400, bcdDevice= 1.00
[  335.433145] usb 7-1.1: New USB device strings: Mfr=17, Product=18, SerialNumber=19
[  335.433149] usb 7-1.1: Product: BillBoard Device
[  335.433151] usb 7-1.1: Manufacturer: Realtek
[  335.433153] usb 7-1.1: SerialNumber: 123456789ABCDEFGH
[  340.671825] usb 7-1.2: new high-speed USB device number 7 using xhci_hcd
[  340.773768] usb 7-1.2: New USB device found, idVendor=17e9, idProduct=4300, bcdDevice=72.51
[  340.773774] usb 7-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[  340.773776] usb 7-1.2: Product: MB16AC
[  340.773777] usb 7-1.2: Manufacturer: DisplayLink
[  340.773778] usb 7-1.2: SerialNumber: 82173223954050M0414
[  341.716479] evdi: [I] (card0) Opened by Task 2738 (DesktopManagerE) of process 2729 (DisplayLinkMana)
[  341.717479] evdi: [I] (card0) Added i2c adapter bus number 19
[  341.717481] evdi: [I] (card0) Connected with Task 2738 (DesktopManagerE) of process 2729 (DisplayLinkMana)
[  341.717484] evdi: [I] (card0) Connector state: connected

**Comment by Kommynct** [comment, marked as answer]
https://gitlab.freedesktop.org/wlroots/wlroots/-/issues/1823

**Comment by nktnet1** [comment]
There's been a recent update to aquamarine (wlroots replacement for Hyprland) which adds support for Displaylink/Evdi:
- https://github.com/hyprwm/aquamarine/pull/25

The stance is still that support is not official, but this has worked for me and many others.
